components/bounties/select-network.tsx

```diff
@@ -57,13 +59,19 @@ export default function SelectNetwork({
       const newQuery = {
         ...query,
         page: "1",
-        networkName: newValue?.value?.name || "all"
+        networkName: newValue?.value?.name || null
       };
 
       push({ pathname: pathname, query: newQuery }, asPath);
     }
   }
 
+  function getSpanClass() {
+    return isProfile
+      ? "caption-small font-weight-medium text-gray-100"
+      : "caption text-gray-500 text-nowrap mr-1 font-weight-normal";
```

Comment:
it's a bit odd that the font weight and colour (100 vs 500) is different depending on if the user is on profile or not -- since this _is_ a component. can you double-check this, please?